content/04.workflows-and-software.md

```diff
@@ -92,7 +93,8 @@ also specifies which Conda channels to search and download programs from. When s
 
 #### Getting started with software management
 
-Using these software management tools within workflow systems is relatively straightforward: first install the manager, then add a system-specific line to each workflow step that specifies information for software installation. With snakemake, for example, each analysis step must contain a link to a software environment (e.g. on of the boxes described in **Figure {@fig:conda_figure}, B**). If the workflow is then executed with the "--use-conda" option, the software specified will be downloaded, installed, used to run that analysis step, and stored for future analysis runs. The specific syntax for using package managers with your chosen workflow system will be available in the documentation.
+Using these software management tools within workflow systems is relatively straightforward: install the manager if needed, then add a system-specific line to each workflow step that specifies information for software installation. With snakemake, for example, add a link to an environment, system module, or container for the required software to the code for that analysis step (conda environment example in **Figure {@fig:conda_figure}, B**).
```

Comment:
```suggestion
Using these software management tools within workflow systems is relatively straightforward: install the manager if needed, then add a line to each workflow step specifying information for software installation. (see conda environment example in **Figure {@fig:conda_figure}, B**).
```

Reply:
I think we can cut it down like that...let's give it a try and we can come back to this section if necessary!